HugeDatabase/Model/Word.swift

```diff
@@ -7,6 +7,6 @@
 
 import Foundation
 
-struct Word: Decodable {
+struct Word: Codable {
```

Comment:
```suggestion
struct Word: Decodable {
```
Je ne suis pas fan que cette structure se conforme à Codable dans la partie Prod. Dans tes tests tu te sers de Encodable pour créer tes données de tests mais Encodable n'est pas nécessaire dans la partie Prod.
A la limite tu pourrais faire une extension de Word dans la target de tests pour quelle se conforme à Encodable, ça serait plus juste.